schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -60,6 +60,15 @@ service SequenceService {
     };
     option (google.api.method_signature) = "name";
   };
+
+    // Attempts a streaming sequence.
+    rpc AttemptStreamingSequence(AttemptStreamingSequenceRequest) returns (google.protobuf.Empty) {
+      option (google.api.http) = {
+        post: "/v1beta1/{name=sequences/*}"
```

Comment:
Per CI: this collides with the `AttemptSequence` URI. So I think you'll want to make this a "custom method" by taking on a custom method verb to the end of the URI. Something like one of the following:
```suggestion
        post: "/v1beta1/{name=sequences/*}:stream"
```
```suggestion
        post: "/v1beta1/{name=sequences/*}:attemptStreamingSequence"
```

Reply:
Ack! We had just noticed right as you commented. We added one of those changes 😄